src/site/content/en/blog/coop-coep/index.md

```diff
@@ -272,18 +300,79 @@ The `endpoints` property specifies the URLs of one or more reporting endpoints.
 The endpoint must accept CORS if it's hosted on a different origin. The browser
 will send reports with a Content-Type of `application/reports+json`.
 
-An example payload looks like this:
+An example COEP report payload when cross-origin resource is blocked looks like
+this:
+
+```json
+[{
+  "age": 25101,
+  "body": {
+    "blocked-url": "https://third-party-test.glitch.me/check.svg?",
+    "blockedURL": "https://third-party-test.glitch.me/check.svg?",
+    "destination": "image",
+    "disposition": "enforce",
+    "type": "corp"
+  },
+  "type": "coep",
+  "url": "https://first-party-test.glitch.me/?coep=require-corp&coop=same-origin&",
+  "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4249.0 Safari/537.36"
+}]
+```
+
+{% Aside 'caution' %}
+`blocked-url` is there for backward compatibility only and will be removed
+eventually.
```

Comment:
Can you link to a Chrome Status entry? If work hasn't been started yet, maybe link to a PR for a spec change. 

This is the first time in six years I've done anything other than replace a link to a spec change. We shouldn't say what we're going to do in the absence of information that says we will do it.

Reply:
I thought this change was subtle enough to skip the intent process. Do we need to send an intent to ship for this change? The merged spec PR is https://github.com/whatwg/html/pull/5848.